descrip.mms

```diff
@@ -38,10 +38,13 @@
 #            NOVA              Just Build The Data General Nova.
 #            PDP1              Just Build The DEC PDP-1.
 #            PDP4              Just Build The DEC PDP-4.
-#            PDP7              Just Build The DEC PDP-7.
-#            PDP8              Just Build The DEC PDP-8.
-#            PDP9              Just Build The DEC PDP-9.
-#            PDP10             Just Build The DEC PDP-10.
+#            PDP6            Just Build The DEC PDP-6.
+#            PDP7            Just Build The DEC PDP-7.
+#            PDP8            Just Build The DEC PDP-8.
+#            PDP9            Just Build The DEC PDP-9.
+#            PDP10           Just Build The DEC PDP-10.
+#            PDP10-KA        Just Build The DEC PDP-10 KA10.
+#            PDP10-KI        Just Build The DEC PDP-10 KI10.

```

Comment:
Indentation changed?

PDP10 could now say "Just build The DEC PDP-10 KS10."

Reply:
It did change when all the new VAX model simulators.  I've fixed that now and added the KS10 wording.  Thanks.